src/docker/eventListener.ts

```diff
@@ -355,6 +357,16 @@ export class EventListener {
       await setContextValue(ContextValues.localSchemaRegistryAvailable, started);
       localSchemaRegistryConnected.fire(started);
     } else if (imageName.startsWith(medusaImage)) {
+      await window.withProgress(
```

Comment:
nit: Up above, strike the "// TODO Patrick: Wait for Medusa log line", todo no longer needed.